stof/doctrine-extensions-bundle/1.2/etc/packages/stof_doctrine_extensions.yaml

```diff
@@ -0,0 +1,21 @@
+stof_doctrine_extensions:
+    default_locale: en_US
+
+    # Activate the extensions you want
+    orm:
+        default:
+            tree: true
+            timestampable: false # not needed: listeners are not enabled by default
+        other:
```

Comment:
this one will cause an issue because the `other` entity manager does not exist on common projects.
Just keep `default` section.